google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/StorageRpc.java

```diff
@@ -553,6 +553,14 @@ TestIamPermissionsResponse testIamPermissions(
    */
   Notification createNotification(String bucket, Notification notification);
 
+  /**
+   * Get the notification with the specified name on the specified object.
+   *
+   * @return the notification object that exist on the bucket.
+   * @throws StorageException upon failure
+   */
+  Notification getNotification(String bucket, String notification);
+
   /**
```

Comment:
Get -> Gets
clarify what is notification name
what happens if no notification exists